sdk/digitaltwins/digitaltwins/src/digitalTwinsClient.ts

```diff
@@ -0,0 +1,851 @@
+   * @param ifMatch Only perform the operation if the entity's etag matches one of the etags provided or * is provided.
+   */
+  public updateRelationship(
+    digitalTwinId: string,
+    relationshipId: string,
+    options?: DigitalTwinsUpdateRelationshipOptionalParams
+  ): Promise<DigitalTwinsUpdateRelationshipResponse> {
+    return this.client.digitalTwins.updateRelationship(digitalTwinId, relationshipId, options);
+  }
+
+  /**
+   * Delete a relationship on a digital twin.
+   *
+   * @param digitalTwinId The Id of the source digital twin.
+   * @param relationshipId The Id of the relationship to delete.
+   * @param ifMatch Only perform the operation if the entity's etag matches one of the etags provided or * is provided.
+   * @returns The http response.
+   */
+  public deleteRelationship(
+    digitalTwinId: string,
+    relationshipId: string,
+    ifMatch: string
+  ): Promise<RestResponse> {
+    var options = <DigitalTwinsDeleteRelationshipOptionalParams>{};
+    options.ifMatch = ifMatch;
+    return this.client.digitalTwins.deleteRelationship(digitalTwinId, relationshipId, options);
+  }
+
+  /**
+   * @internal
+   * @ignore
+   * Deals with the pagination of {@link listRelationships}.
+   *
+   * @param {PageSettings} continuationState An object that indicates the position of the paginated request.
+   * @param {DigitalTwinsListRelationshipsOptionalParams} [options] Common options for the iterative endpoints.
+   *
+   */
+  private async *listRelationshipsPage(
+    digitalTwinId: string,
+    options: DigitalTwinsListRelationshipsOptionalParams,
+    continuationState: PageSettings
+  ): AsyncIterableIterator<DigitalTwinsListRelationshipsResponse> {
+    if (continuationState.continuationToken == null) {
+      const optionsComplete: DigitalTwinsListRelationshipsOptionalParams = {
+        maxresults: continuationState.maxPageSize,
+        ...options
+      };
+      const listRelationshipResponse = await this.client.digitalTwins.listRelationships(
+        digitalTwinId,
+        optionsComplete
+      );
+      continuationState.continuationToken = listRelationshipResponse.nextLink;
+      yield listRelationshipResponse;
+    }
+    while (continuationState.continuationToken) {
+      const listRelationshipResponse = await this.client.digitalTwins.listRelationshipsNext(
+        continuationState.continuationToken,
+        options
+      );
+
+      continuationState.continuationToken = listRelationshipResponse.nextLink;
+      yield listRelationshipResponse;
+    }
+  }
+
+  /**
+   * @internal
+   * @ignore
+   * Deals with the iteration of all the available results of {@link listRelationships}.
+   * @param {DigitalTwinsListRelationshipsOptionalParams} [options] Common options for the iterative endpoints.
+   */
+  private async *listRelationshipsAll(
+    digitalTwinId: string,
+    options: DigitalTwinsListRelationshipsOptionalParams
+  ): AsyncIterableIterator<any> {
+    const f = {};
```

Comment:
what is this argument?

Reply:
Removed

---
In reply to: [447262020](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447262020) [](ancestors = 447262020)